types/cacheable-request/index.d.ts

```diff
@@ -33,6 +31,8 @@ interface CacheableRequest {
 }
 
 declare namespace CacheableRequest {
+    type StorageAdapter = Store<any>;
```

Comment:
Please don't hard-code the `any` type parameter in the `StorageAdapter` type to allow users of this type to use this parameter to correctly align `get()` and `set()` signatures of the `StorageAdapter` implementation:

```suggestion
    type StorageAdapter<TValue> = Store<TValue>;
```

Don't forget to set this parameter to `any` in the CacheableRequest constructor argument.

Reply:
Isn't the value type determined by the library? It's not as if the caller can meaningfully declare any type they choose; for their type to be helpful it needs to conform to the library's usage:

```js
{
  cachePolicy: response.cachePolicy.toObject(),
  url: response.url,
  statusCode: response.fromCache ? revalidate.statusCode : response.statusCode,
  body
}
```

If we don't use `Store<any>` it seems like it would be better to add an explicit type for the cache value.

Reply:
If you would add an explicit type for the cache value, that would be great! Otherwise, stick with what you've done so far.

Reply:
Thanks, I'm happy to address that in a follow-on.